homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -44,9 +45,28 @@ class LaCrosseSensorEntityDescription(
     """Description for LaCrosse View sensor."""
 
 
-def get_value(sensor: Sensor, field: str) -> float:
+def get_value(sensor: Sensor, field: str) -> float | int | str:
```

Comment:
Is there a compelling reason why you'd want to return an `int`? A `float` is a perfectly fine numeric value, and you're not losing much by returning `1.0` instead of `1`